### ISSUE TITLE
Limits?

### ISSUE THREAD

**Issue by michevnew** [MicrosoftDocs/office-docs-powershell#3288]
Can you please specify the limits for allowed/blocked domains, allowed/blocked senders, etc?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b7e1bc61-af1f-a9c1-72fe-ece163efc0f6
* Version Independent ID: 2ba3f316-1872-5670-22b3-94f1c4cfcb0f
* Content: [Set-HostedContentFilterPolicy (exchange)](https://docs.microsoft.com/en-us/powershell/module/exchange/antispam-antimalware/set-hostedcontentfilterpolicy?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/antispam-antimalware/Set-HostedContentFilterPolicy.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/antispam-antimalware/Set-HostedContentFilterPolicy.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by yogkumgit** [comment]
@officedocsbot assign @yogkumgit

**Comment by dariomws306** [comment]
Hi @michevnew, thank you for your feedback.

Please check the article I have added in the last section (related links). It will be public in some days.

**Comment by yogkumgit** [comment]
@chrisda please merge PR #3348

**Comment by yogkumgit** [comment]
@dariomws306  Thank you very much for the contribution and updating the documentation with PR. @michevnew Hope this update is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by yogkumgit** [comment]
@officedocsbot close

**Comment by michevnew** [comment]
Sorry to reopen this, but the updated document simply adds a link to the per-mailbox safe sender/blocked sender limits article. What I would like to see is the tenant-wide limits mentioned somewhere, such as the limit of entries (size) for the BlockedSenderDomains attribute, etc.

**Comment by dariomws306** [comment]
Hi @michevnew,

I think you are looking for this:
https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-protection-service-description/exchange-online-protection-limits

Right?

**Comment by chrisda** [comment]
@michevnew, whenever you think of limits for features in the cloud, think of the Office 365 Service Descriptions. I found the information in the [Exchange Online Protection Limits](https://docs.microsoft.com/office365/servicedescriptions/exchange-online-protection-service-description/exchange-online-protection-limits) topic.

Apparently the limit is 1024 for allowed senders and 1024 blocked senders, although I can't tell if that's 1024 for allowed individual senders and another 1024 for allowed domains, or if the limit is 1024 for both (same for blocked).

I added a link to that service description topic to New/Set-HostedContentFilterPolicy. I'm hesitant to mention the actual numbers there; the service description should be the one source of truth for limits like this, because they could change in the future.

Thanks for the feedback.
